README.md

```diff
@@ -21,41 +22,48 @@ Cognito Quickstart
 ##### Install the required tools
 * Create an AWS account
 * Install [npm](https://www.npmjs.com/)
-* Run ```npm install``` at the project's root level
 * [Install or update your aws cli](http://docs.aws.amazon.com/cli/latest/userguide/installing.html) 
+* [Install or update your eb cli](http://docs.aws.amazon.com/elasticbeanstalk/latest/dg/eb-cli3-install.html) 
 * [Install angular-cli](https://github.com/angular/angular-cli)
 
-##### This will install the AWS resources
-* ```cd aws``` 
-* ```./createResources.sh```
-
 
 # Getting the code
 ```
 # Clone it from github
 git clone --depth 1 git@github.com:awslabs/aws-cognito-angular2-quickstart.git
 ```
 ```
-# Install the NPM packages
-npm install
+# Install the AWS resources and deploy your application to either Elastic Beanstalk or S3
+cd aws
+./createResources.sh
 ```
+
+# Elastic Beanstalk
 ```
 # Run the app in dev mode
-npm start
+npm install; npm run build; npm start
 ```
 ```
-# Build the project and sync the output with the S3 bucket
-ng build; cd dist; aws s3 sync . s3://budilov-cognito/
+# Commit your changes and deploy it to your environment
+git add .
+git commit
+eb deploy
```

Comment:
`eb deploy` will run but without the changes. If you want the  changes to be deployed to the environment, you will have to add & commit before running this command